### ISSUE TITLE
Feature Request: allow configuring to skip checkstyle analysis some directories/files/projects without changing the configuration file

### ISSUE THREAD

**Issue by mtomim** [jdneo/vscode-checkstyle#313]
The plugin works great, but I really want to be able to configure projects/files/directories in which checkstyle doesn't apply and avoid coloring everything in red, without modifying the configuration file itself, nor having to change everything in the massive old unclean project. If it's possible via the parameter "java.checkstyle.properties" for example, I want to see some examples in README.

For example:
```javascript
{
  "java.checkstyle.ignore": [ /* file pattern array */ ]
}
```
The word "ignore" may well be "skip".

Thanks in advance for the consideration!

![image](https://user-images.githubusercontent.com/45530298/116416824-fbbd7880-a83a-11eb-8bce-09f3d3d08d0f.png)

**Comment by Keetz** [comment]
This would be a much appreciated feature.

I have a bunch of generated classes which doesn't quiet match my projects overall checkstyle. It would be sweet to just ignore that folder containing all those generated classes, and get rid of the false-positive errors.
